analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -1184,12 +1185,10 @@ def extend_compilation_database_entries(compilation_database):
                 for source_file in source_files:
                     new_entry = dict(entry)
                     new_entry['file'] = source_file
-                    entries.append(new_entry)
```

Comment:
Why is it useful to build this list?

Reply:
Oh, sorry, for some reason I though it was a "yield -> append" change, but it's just reverse :D Sorry.